contrib/stage_docs.sh

```diff
@@ -2,6 +2,8 @@
 
 set -ex
 
+pushd "$(dirname "$0")"
```

Comment:
[q] does this script leave you in this directory? might be safe to just `popd` when you no longer want to be in this directory